app/controllers/bulk_taggings_controller.rb

```diff
@@ -0,0 +1,37 @@
+class BulkTaggingsController < ApplicationController
+  def new
+    render :new, locals: { results: [] }
+  end
+
+  def search
+    if search_response.successful?
```

Comment:
If the API does not respond properly (i.e. with code 200). I'm imagining a timeout (we saw some locally). If we don't do this we get a 500 error...